src/components/PlayIcon.js

```diff
@@ -0,0 +1,10 @@
+import Icon from '@mdi/react';
+import {mdiPlay} from '@mdi/js';
+
+export default function PlayIcon() {
+  return (
+    <>
+      <Icon path={mdiPlay} title="Play" size={0.7} color="white" />
+    </>
+  );
```

Comment:
I think you can remove the fragment here:
```suggestion
  return <Icon path={mdiPlay} title="Play" size={0.7} color="white" />;
```